equipment_handler/src/adcs_handler.c

```diff
@@ -1300,7 +1304,12 @@ ADCS_returnState ADCS_save_img(uint8_t camera, uint8_t img_size) {
 ADCS_returnState ADCS_set_magnetorquer_output(xyz16 duty_cycle) {
     uint8_t command[7];
     command[0] = SET_MAGNETORQUER_OUTPUT_ID;
-    memcpy(&command[1], &duty_cycle, 6);
+    command[1] = (duty_cycle.x) & 0x00FF;
+    command[2] = (duty_cycle.x >> 8) & 0x00FF;
+    command[3] = (duty_cycle.y) & 0x00FF;
+    command[4] = (duty_cycle.y >> 8) & 0x00FF;
+    command[5] = (duty_cycle.z) & 0x00FF;
+    command[6] = (duty_cycle.z >> 8) & 0x00FF;
```

Comment:
@arrooney do these bitwise ANDs seem necessary to you? I'm not sure

Reply:
And if they are necessary cannot they be shortened to 0xFF?

Reply:
well I mean why not stick with the memcpy?

Reply:
In this case the memcpy might not be ideal because the layout of the struct may change without considering this section of the code but the fields should stay the same

Reply:
Actually these right shifts on signed integers make me a bit nervous. Might be safer just to leave the 0x00FF

https://stackoverflow.com/questions/8422424/can-you-control-what-a-bitwise-right-shift-will-fill-in-c